pkg/local_object_storage/metabase/graveyard.go

```diff
@@ -261,6 +262,52 @@ func (db *DB) DropGraves(tss []TombstonedObject) error {
 	})
 }
 
+// DropExpiredTSMarks run through the graveyard and drops tombstone marks with
+// tombstones whose expiration is _less_ than provided epoch.
+// Returns number of marks dropped.
+func (db *DB) DropExpiredTSMarks(epoch uint64) (int, error) {
+	db.modeMtx.RLock()
+	defer db.modeMtx.RUnlock()
+
+	if db.mode.NoMetabase() {
+		return -1, ErrDegradedMode
+	} else if db.mode.ReadOnly() {
+		return -1, ErrReadOnlyMode
+	}
+
+	var counter int
+
+	err := db.boltDB.Update(func(tx *bbolt.Tx) error {
+		bkt := tx.Bucket(graveyardBucketName)
+		c := bkt.Cursor()
+		k, v := c.First()
+
+		for k != nil {
+			if binary.LittleEndian.Uint64(v[addressKeySize:]) < epoch {
```

Comment:
i'd still check `v` length explicitly to avoid unclear panic, especially since the suffix was not always there

Reply:
> to avoid unclear panic

and what should such code do then? err? warn print? skip? since we have migration now and db versioning, this situation unexpected and never is a normal situation

Reply:
panic with clear report

Reply:
should such code (and many more) have clear panic too: https://github.com/nspcc-dev/neofs-node/blob/b135a0efe9c6e38ce69f53f9dd1c191d62f218bd/pkg/local_object_storage/metabase/util.go#L248-L258
?

Reply:
any panic should be as much informative as possible preferably